docs/source/library.rst

```diff
@@ -10,4 +10,7 @@ RCCL
 Introduction
 ------------
 
-The RCCL is an AMD port of NCCL.
+The ROCm Collective Communication Library (RCCL) is a stand-alone library which provides multi-GPU and multi-node collective communication primitives optimized for AMD GPUs and can be easily integrated into applications. 
```

Comment:
Please write add a separate sentence for 
and can be easily integrated into applications.